docs(admin/workspace-proxies): add instructions to run using the Docker image (#7915)
coder/coder: docs/admin/workspace-proxies.md

coder wsproxy server
```

> ### Running from the Docker image

Reviewer: ```suggestion
### Running in Docker
```

> Modify the default entrypoint to run the workspace proxy.

Reviewer: ```suggestion
Modify the default entrypoint to run a workspace proxy server instead of a regular Coder server.
```

#### Docker Compose

> change the provided [`docker-compose.yml`](https://github.com/coder/coder/blob/main/docker-compose.yaml) file to include the workspace proxy.

Reviewer: ```suggestion
Change the provided [`docker-compose.yml`](https://github.com/coder/coder/blob/main/docker-compose.yaml) file to include a custom entrypoint:
```

```diff
  image: ghcr.io/coder/coder:${CODER_VERSION:-latest}
+ entrypoint: /opt/coder wsproxy server
```

#### Docker run

```bash
docker run --rm -it --entrypoint /opt/coder ghcr.io/coder/coder:latest wsproxy server
```

#### Custom Dockerfile

```Dockerfile
FROM ghcr.io/coder/coder:latest
ENTRYPOINT ["/opt/coder", "wsproxy", "server"]
```

### Selecting a proxy

Users can navigate to their account settings to select a workspace proxy. Workspace proxy preferences are cached by the web browser. If a proxy goes offline, the session will fall back to the primary proxy. This could take up to 60 seconds.

![Workspace proxy picker](../images/admin/workspace-proxy-picker.png)

> 
> ```
> 
> ```
> 
> ```
> 
> ```

Reviewer: ```suggestion
```
